.gitattributes

```diff
@@ -0,0 +1 @@
+alternate_formats/* -diff
```

Comment:
If we're checking that the contents match the human-readable-and-diffable YAML file, that seems fine. We can optimize for performance on the JSON output.